smoke/tests/tool/layer.go

```diff
@@ -44,15 +44,20 @@ func (l *Layer) CreateFile(t *testing.T, name string, data []byte) {
 	require.NoError(t, err)
 }
 
-func (l *Layer) CreateLargeFile(t *testing.T, name string, sizeMB int) {
+func (l *Layer) CreateLargeFile(t *testing.T, name string, sizeGB int) {
 	f, err := os.Create(filepath.Join(l.workDir, name))
 	require.NoError(t, err)
 	defer func() {
 		f.Close()
 	}()
 
-	for b := 1; b <= sizeMB; b++ {
-		_, err := f.Write(bytes.Repeat([]byte{byte(b)}, 1024*1024))
+	buf := make([]byte, 0x1000)
+	for i := 0; i < len(buf); i++ {
+		buf[i] = byte(i)
+	}
+
```

Comment:
How about using `trunc()` to set file size instead of writing the whole file?

Reply:
Creating a 5G file for tests is challenging:)

Reply:
> Creating a 5G file for tests is challenging:)

In practice, 5G file is very slow in pack, merge and verify. By `LargerFileMaker`, texture provides the ability to create large file. No large file default. 
So far, it's only used in this case: large file in prefetch.

Reply:
Can we use `rand.Reader` and `io.CopyN` here?